test/fw/bin/pbs_loganalyzer

```diff
@@ -151,7 +153,7 @@ if __name__ == '__main__':
                    "re-conditional=", "estimated-info-only", "silent",
                    "db-type=", "db-access="]
         opts, args = getopt.getopt(sys.argv[1:], shortopt, longopt)
-    except:
+    except BaseException:
```

Comment:
The correct answer for blanket excpets is to determine what exceptions are expected.  Doing this is no better than a blanket except.  BaseException includes things like KeyboardInterrupt which is undesired.  Until we go through to fix all of these at once, I'd leave them alone.